docs/source/sections/data.rst

```diff
@@ -4,7 +4,11 @@
 Working with data and querying your results
 *******************************************
 
-In this section of the tutorial we will focus on how to organise and explore the data in an AiiDA database, based on a previously created database  for this tutorial.
+.. todo::
+
+   Probably requires update.
```

Comment:
So you didn't test this section yet, right? 🙃

Reply:
Just quickly executed the commands in the section and then ran the whole solution notebook. Seems we have to remove the duplicate groups made with this command:

```console
$ verdi group copy tutorial_lda tutorial/lda/basic
$ verdi group copy tutorial_pbe tutorial/gga/pbe
$ verdi group copy tutorial_pbesol tutorial/gga/pbesol
```

Else the final plot won't work (it expects 3 pseudopotential groups, not 6).